tests/drivers/led/led_api/src/test_led_api.c

```diff
@@ -14,7 +14,6 @@
 
 #if DT_NODE_HAS_STATUS(DT_ALIAS(led_controller_0), okay)
 #define LED_CTRL_NODE_ID	DT_ALIAS(led_controller_0)
-#define LED_CTRL_DEV_NAME	DT_LABEL(LED_CTRL_NODE_ID)
 #else
 #error "LED controller device not found"
 #endif
```

Comment:
let's delete all that block for consistency with other samples/tests and use DT_ALIAS() below